lib/index.js

```diff
@@ -53,12 +53,13 @@ class Rooftop {
       }
 
       if (transformFn === true) transformFn = transform
-      if (transformFn === false) transformFn = (x) => x
+      else if (transformFn === false) transformFn = (x) => x
 
       return this.client[name].get(options)
-        .then((p) => { return W.map(p, transformFn) })
-        .tap((res) => { m[name] = res })
-        .yield(m)
+        .then((p) => W.map(p, transformFn))
+        .then((v) => m[name] = v)
+        .then(() => m)
```

Comment:
`yield` is more efficient here. and i think standard will complain if you return an assignment, hence the brackets